checker/src/main/java/org/checkerframework/checker/index/qual/EnsuresLTLengthOf.java

```diff
@@ -52,6 +53,7 @@
 @Target({ElementType.METHOD, ElementType.CONSTRUCTOR})
 @PostconditionAnnotation(qualifier = LTLengthOf.class)
 @InheritedAnnotation
+@Repeatable(EnsuresLTLengthOfMultiple.class)
```

Comment:
By convention, the repeating annotation is given a plural name, which here would be `EnsuresLTLengthOfs`.
I agree that's a bit hard to read.

Another possibility is to use an [inner annotation named `@EnsuresLTLengthOf.List`](https://dzone.com/articles/repeatable-annotations-in-java-8-1).  I like that convention and propose that we adopt it.
@wmdietl @smillst What do you think?

Reply:
I like `@EnsuresLTLenghtOf.List`, too.

Reply:
Done this convention in #2625